src/js/background/background.js

```diff
@@ -38,11 +39,34 @@ async function getAliasesFromServer(method = "GET", body = null, opts=null) {
     mode: "same-origin",
     method,
     headers: headers,
-    body,
   });
 
   answer = await response.json();
   
+  if (opts.subdomainSet) {
+    
+    const masks = new Array();
+    masks.push(...answer);
+
+    const domainResponse = await fetch(apiMakeDomainAddressesURL, {
+      mode: "same-origin",
+      method,
+      headers: headers,
+    });
+
+    domainMasks = await domainResponse.json();
+    masks.push(...domainMasks);
+    masks.sort((a, b) => (a.created_at < b.created_at ? 1 : -1));
+
+    browser.storage.local.set({ relayAddresses: masks });
```

Comment:
You know the add-on code better than I do, so this is more of a check, but: given that this property is called `relayAddresses` (usually used for random masks), I'm wondering how certain we are that shoving custom masks in there will not cause problems in other code paths that assume this to only contain random ones?

Reply:
Fair point. I'll confirm the other spots this is used. I was going to clean up the calls so that they all use this one method. The only other spot where these masks would be listed is in the context menu, for now _(the upcoming popup refactor will include this function too)._

Reply:
Welp, great news here: I think I solved #231 #238 and #233. Now that we're querying ALL the masks, it's adding in the actual last 5 used, not just random.  More to test, but it's a start! 

Revised in 3e52dbdf75aa03dc0f96d89c252861af3b081fc8

Reply:
Nice!